app/models/purpose.rb

```diff
@@ -40,7 +41,9 @@ class Purpose < ApplicationRecord
 
   before_validation :set_default_barcode_prefix
 
+  # rubocop:todo Rails/UniqueValidationWithoutIndex
   validates :name, format: { with: /\A\w[\s\w.\-]+\w\z/i }, presence: true, uniqueness: { case_sensitive: false }
+  # rubocop:enable Rails/UniqueValidationWithoutIndex
 
```

Comment:
Weirdly it was the removal of the brackets around 'plate_purposes' above that let rubocop detect this. There are a load of them in the rubocop_ignore file as well. SHould be easy enough to fix.